setup.py

```diff
@@ -22,7 +22,7 @@
     "cachetools>=2.0.0,<5.0",
     "pyasn1-modules>=0.2.1",
     # rsa >= 4.1 no longer supports python 2 https://github.com/sybrenstuvel/python-rsa/issues/152
-    'rsa<4.1; python_version < "3"',
```

Comment:
Thank you! I'd suggest: 
- check that rsa 4.5 does not cause a breakage  on Py2
- set this to '<4.6', since 4.5 should be the final version supporting Py2 as per https://github.com/sybrenstuvel/python-rsa/issues/152#issuecomment-643470233
- Update the comment in line 24.

Reply:
cc: @busunkim96